lib/services/webSiteManagement2/lib/models/address.js

```diff
@@ -1,9 +1,5 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- * 
- * Code generated by Microsoft (R) AutoRest Code Generator 0.15.0.0
+ * Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
```

Comment:
Could you update the file with the new format for `autorest_options`? I didn't test it, I think if you try to use the old string format for `autorest_options` now that it has to be a dict, results are unexpected.